src/main/msp/msp.c

```diff
@@ -4051,10 +4050,29 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
                     textVar = currentControlRateProfile->profileName;
                     break;
 
+                case MSP2TEXT_CUSTOM_MSG_0:
+                case MSP2TEXT_CUSTOM_MSG_0 + 1:
+                case MSP2TEXT_CUSTOM_MSG_0 + 2:
+                case MSP2TEXT_CUSTOM_MSG_0 + 3:
+                case MSP2TEXT_CUSTOM_MSG_0 + 4:
+                case MSP2TEXT_CUSTOM_MSG_0 + 5:
+                case MSP2TEXT_CUSTOM_MSG_0 + 6:
+                case MSP2TEXT_CUSTOM_MSG_0 + 7:
+                    {
+                        unsigned msgIdx = textType - MSP2TEXT_CUSTOM_MSG_0;
+                        if (msgIdx < OSD_CUSTOM_MSG_COUNT) {
+                            textVar = pilotConfigMutable()->message[msgIdx];
```

Comment:
Why handle 8 messages and then limit to four. Do we need to accommodate a variable number of messages, up to 8? Why not encode this as a message with two arguments, the message id and the message rather than having 8 codes?

Reply:
it is consistent with other names (all are message ids for MSP2_SET_TEXT)